rust/agama-lib/share/profile.schema.json

```diff
@@ -105,6 +105,24 @@
                   }
                 }
               },
+              "bond": {
+                "type": "object",
+                "description": "Bonding configuration",
+                "additionalProperties": false,
+                "properties": {
+                  "options": {
+                    "type": "string"
+                  },
+                  "ports": {
+                    "type": "array",
+                    "items": {
+                      "description": "A list of the ports to be bonded",
```

Comment:
Maybe we need to be more precise here:
1) Is it a connection name or interface name
2) What happen if it is a connection name, but no connection with this name exists (defaults)